requirements.txt

```diff
@@ -1,24 +1,6 @@
-## The following combination still work, but later versions of 1.x were broken
-# due to their upstream package brought in a breaking change
-# https://github.com/pallets/markupsafe/issues/286
-# Flask==1.1.4
-# werkzeug==1.0.1
-# Jinja2==2.11.3
-# MarkupSafe==2.0.1
-#
-# Since Flask family do not use semantic versioning, we remove our upper bound here
 Flask>=2
+Flask-Session2
```

Comment:
Flask-Session doesn't seem to be maintained currently, but Flask-Session2 has a high amount of activity. (I did try getting rid of the dependency entirely, but unfortunately, since the cookie is so large, we can't get away with using Flask's built-in cookie sessions)

Reply:
Thanks for bringing Flask-Session2 to our attention. However, [its history](https://pypi.org/project/flask-session2/#history) is still short. Perhaps, we can keep the old Flask-Session for now, but adding a comment to say "you may consider switch to Flask-Session2 (https://...)".

If your next PR can get rid of the dependency entirely, it would be great!